frontend/public/components/utils/headings.tsx

```diff
@@ -120,7 +120,9 @@ export const PageHeading = connectToModel((props: PageHeadingProps) => {
     (hasButtonActions || hasMenuActions || customActionMenu) &&
     hasData &&
     !_.get(data, 'metadata.deletionTimestamp');
-  const resourceStatus = hasData && getResourceStatus ? getResourceStatus(data) : null;
+  const errCode = _.get(obj, ['loadError', 'json', 'code'], null);
+  const resourceStatus =
+    errCode === 404 ? 'Unknown' : hasData && getResourceStatus ? getResourceStatus(data) : null;
```

Comment:
I don't think this is the right approach to this problem . We need to dig deeper as to why the pool is not getting deleted ? Should we be redirecting to the pool's list page after deleting the resource ? 

It is a backend issue if the pool is not getting deleted properly.

Reply:
I can confirm that the pool is being deleted properly, but the response that we get here is not updated to "Unknown" (or even "Terminating") in case of deletion.

Reply:
> but the response that we get here is not updated to "Unknown" (or even "Terminating") in case of deletion.

Setting Status.Phase is up to the backend team.  If pool is getting deleted properly then no issues.  Please redirect back to pool's list .